dashboard/test/ui/features/blockTrashing.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Blocks can be trashed in certain circumstances
+
+Background:
+  Given I am on "http://learn.code.org/s/course2/stage/19/puzzle/2"
+  # In this stage:
+  # block 18 is the "when run" block
+  # block 19 is an undeletable "set color" block
+  # block 24 is a deletable "turn right" block
```

Comment:
I think that it might be the case that the more correct way of writing these tests is to do something like
"When I drag block friendlyNameFoo to offset bar" and then in our ruby code define friendlyNameFoo. i.e. the feature files should be written in pretty readable English. How you have it write now is obviously how most of our existing tests work however.

I don't know if there's a way to alias, though you could accomplish something like this by having a the "blocks is a child of block" step also take strings as inputs, and give it a mapping of string to block number.